### DISCUSSION TITLE
Intro/Refresher to FRC Programming

### DISCUSSION THREAD

**Discussion by JooshO** [frc5567/2023-Robot#199] [Team Posts]
Hey everyone,
I'm planning on running a somewhat brief, 1 to 1.5 ish hour introduction or refresh session on FRC programming. Specifically, I would like to focus on data types, classes, the way we write code, and some of the common WPILib classes. If you are interested in coming, please fill out the whenisgood linked below so I can get an idea for when would work for everybody. If you know anybody who is interested in learning who is not on Github, please forward this message to them so we can involve everybody who wants to be there.

[https://whenisgood.net/8jrwa27](https://whenisgood.net/8jrwa27)

**Comment by JooshO** [comment]
Based on the responses I have received so far, I am going to schedule the meeting for 2PM on Saturday the 2nd. The zoom link will be attached below. Even if you did not fill out the whenisgood, you are still welcome to come. I appreciate everyone who did fill that out.

If you know anybody who is not on Github who would be interested in going, please forward them the zoom link and information.

[https://zoom.us/j/91470076435?pwd=Zk03dzNVR1pNYUU2ZzM4NkdER0lxdz09](https://zoom.us/j/91470076435?pwd=Zk03dzNVR1pNYUU2ZzM4NkdER0lxdz09)

Meeting ID: 914 7007 6435
Passcode: 0aD0Ei

**Comment by JooshO** [comment]
Here is the link to a document walking through Git and VSCode installation, if you need more detail or help feel free to reach out to me or a mentor that is available.

[Link](https://docs.google.com/document/d/1axB8mP_1fp1kZ5W4wzsPzkWifgF--jJoDHOVsoQ7fKs/edit?usp=sharing)

**Comment by coverbeek** [comment]
On behalf of the mentors, I want to say thanks to Josh for organizing this. Hopefully those who participated found it valuable. 

Feel free to contact any of us if you have questions, or post here and we can answer so everyone can benefit.
